recipes/openai/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "openai" %}
+{% set version = "0.3.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f514098dece360dcee8fe1b33eed876955af7f2e3028362a54f00f8522b71408
+
+build:
+  number: 0
+  skip: true   # [py==30 or py==31 or py==32 or py==33 or py<27]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - requests >=2.20
```

Comment:
I'm not completely sure about this, `install_requires` [lists](https://github.com/openai/openai-python/blob/f00f489455ccbf4feb4bcf65f89d497fc4dd31ff/setup.py#L15-L18) more than one `requests` but when installing it with pip, the output is the following:

```zsh
Collecting openai
  Downloading openai-0.3.0.tar.gz (157 kB)
     |████████████████████████████████| 157 kB 5.2 MB/s 
Collecting requests>=2.20
  Using cached requests-2.25.1-py2.py3-none-any.whl (61 kB)
Collecting urllib3<1.27,>=1.21.1
  Using cached urllib3-1.26.3-py2.py3-none-any.whl (137 kB)
Collecting idna<3,>=2.5
  Using cached idna-2.10-py2.py3-none-any.whl (58 kB)
Collecting chardet<5,>=3.0.2
  Using cached chardet-4.0.0-py2.py3-none-any.whl (178 kB)
Requirement already satisfied: certifi>=2017.4.17 in /opt/anaconda3/envs/test-pip/lib/python3.9/site-packages (from requests>=2.20->openai) (2020.12.5)
Building wheels for collected packages: openai
  Building wheel for openai (setup.py) ... done
  Created wheel for openai: filename=openai-0.3.0-py3-none-any.whl size=171249 sha256=fe48d721b05b30cc0e8a437ab84195a40487af5b54cfa85cfa73a19e1d1dd580
  Stored in directory: /Users/username/Library/Caches/pip/wheels/21/1b/1c/f4b826a625463f325c6ee6feab17a525379f885f54e6b2230d
Successfully built openai
Installing collected packages: urllib3, idna, chardet, requests, openai
Successfully installed chardet-4.0.0 idna-2.10 openai-0.3.0 requests-2.25.1 urllib3-1.26.3
```

Reply:
Am I correct in understanding that for `python <3` `requests[security]` is needed and for anything else just `requests`? Then I'd just make this `python3` only.